templates/keycloak/files/.platform.app.yaml

```diff
@@ -0,0 +1,83 @@
+name: 'app'
+
+type: 'java:11'
+
+variables:
+    env:
+        KC_VERSION: 8.0.2
+        JBOSS_HOME: /app
+        JDBC_MARIADB_VERSION: 2.2.3
+        LAUNCH_JBOSS_IN_BACKGROUND: 1
+        #DB_VENDOR: mariadb
+hooks:
+    build: |
+        set -e
+
+        if [ ! -d "$PLATFORM_CACHE_DIR"/keycloak-$KC_VERSION ]; then
```

Comment:
1. This build hook looks complex enough it would be easier to read if moved to a separate shell script.
2. It's downloading several things, which means it would be better to leverage the build cache to avoid doing so repeatedly on every build.  That would likely speed up the build considerably.  Let's do that.

Reply:
@Crell it has 21 lines, I would like to keep on this file for while, also the condition is to avoid download repeatedly

Reply:
... I am blind and moving too fast these days. :frowning: You're right, it's already using the cache, on this very line.